dev/com.ibm.websphere.appserver.features/visibility/public/jaxb-2.3/com.ibm.websphere.appserver.jaxb-2.3.feature

```diff
@@ -18,7 +19,7 @@ Subsystem-Name: Java XML Bindings 2.3
 -features=\
   com.ibm.websphere.appserver.classloading-1.0
 -bundles=\
-  com.ibm.websphere.javaee.activation.1.1; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.activation:activation:1.1", \
+  com.ibm.websphere.javaee.activation.1.1; require-java:="9"; location:="dev/api/spec/,lib/"; mavenCoordinates="javax.activation:activation:1.1", \
```

Comment:
this change can be undone, it was correct before (always providing javax.activation)